lib/client.js

```diff
@@ -247,31 +257,46 @@ Client.prototype.removeSeedHost = function (hostname, port) {
  *
  * @example
  *
+ * const Aerospike = require('aerospike')
+ * const Key = Aerospike.Key
+ * 
+ * // INSERT HOSTNAME AND PORT NUMBER OF AEROSPIKE SERVER NODE HERE!
+ * var config = {
+ *   hosts: '192.168.33.10:3000',
+ *   // Timeouts disabled, latency dependent on server location. Configure as needed.
+ *   policies: {
+ *     batch : new Aerospike.BatchPolicy({socketTimeout : 0, totalTimeout : 0}),
+ *   }
+ * }
+ * 
  * var keys = [
  *   new Key('test', 'demo', 'key1'),
  *   new Key('test', 'demo', 'key2'),
  *   new Key('test', 'demo', 'key3')
  * ]
- *
- * client.batchExists(keys, function (error, results) {
- *   if (error) {
- *     // handle failure
- *   } else {
- *     results.forEach(function (result) {
+ * 
+ * Aerospike.connect(config, (error, client) => {
```

Comment:
Just a suggestion: If you are anyway updating all the examples, then it would be better to replace all usage of callback functions with async/await, rather than just replacing regular functions with arrow functions.